deployments/vacanciesmanage.yml

```diff
@@ -0,0 +1,235 @@
+trigger: none
+pr: none
+
+variables:
+- group: Release Management Resources
+- group: RELEASE das-apim-endpoints
+- name: OuterApiObject
+  value: $(VacanciesManageOuterApiObject)
+
+resources:
+  repositories:
+  - repository: self
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.3.31
+    endpoint: SkillsFundingAgency
+  - repository: das-platform-automation
+    type: github
+    name: SkillsFundingAgency/das-platform-automation
+    ref: refs/tags/4.4.1
```

Comment:
```suggestion
    ref: refs/tags/4.5.0
```
Can be latest tag

Reply:
https://github.com/SkillsFundingAgency/das-platform-automation/releases

Reply:
👍🏻 thats done